### PR TITLE
Refactor/list

### PR DESCRIPTION
**Pull request by ChristianJoe**
Due to FB developer announcement, classic list won't work anymore.
Refactor to generic template with up to 10 carrousel elements. 
Merge to master in order to present to client for further development, in particular UI questions need to be handled.

Share function seems to be fine, since just urlButton is used.

### CODE REVIEW

File: handler/payloadReport.js [hunk 1]
```diff
@@ -7,6 +7,10 @@ import translations from "../assets/translations";
 const handler = async function(chat, payload) {
     const url = `${urls.report(payload.report)}`;
 
+    if ('language' in payload) {
```
Comment (jh0ker) [inline handler/payloadReport.js]: why is this needed?

File: handler/payloadLatestReports.js [hunk 2]
```diff
@@ -36,7 +36,7 @@ export default async function(chat) {
             translateReport(report, chat).text,
             [ buttonPostback(
                 chat.getTranslation(translations.readReport),
-                { action: 'report', report: report.id }
+                { action: 'report', report: report.id, 'language': chat.language }
```
Comment (jh0ker) [inline handler/payloadLatestReports.js]: why is this needed?

File: lib/facebook.js [hunk 3]
```diff
@@ -181,18 +181,13 @@ export class Chat {
                 attachment: {
                     type: 'template',
                     payload: {
-                        'template_type': 'list',
-                        'top_element_style': topElementStyle,
+                        'template_type': 'generic',
```
Comment (jh0ker) [inline lib/facebook.js]: this is not acceptable. we need a new function and all uses need to be switched to the new function. the original `sendList` needs to stay or be deleted if it's not used anymore.

File: handler/payloadLatestReports.js (hunk 2 above)
Comment (ChristianJoe) [reply handler/payloadLatestReports.js]: Since not every report exists in every language, you can get a list of all reports with language A, change settings to language B, go back to the open list of reports and open one report with no translation in B and receive an error.
Figured this out while testing the different lists we use.

Comment (jh0ker) [reply handler/payloadLatestReports.js]: ohh that makes sense, thanks :)

File: lib/facebook.js (hunk 3 above)
Comment (ChristianJoe) [reply lib/facebook.js]: We now have a function `sendGenericList` used in all list cases using `listElement` as its elements.
